commons/passive/garmin/garmin_generic_intensity.avsc

```diff
@@ -0,0 +1,43 @@
+{
+   "name":"GarminGenericIntensity",
+   "type":"record",
+   "namespace":"org.radarcns.passive.garmin",
+   "doc":"Intensity data from Garmin device.",
+   "fields":[
+        {
+            "name":"time",
+            "type":"double",
+            "doc":"Last updated time."
+         },
+         {
+            "name":"timeReceived",
+            "type":"double",
+            "doc":"Device receiver timestamp in UTC (s)."
+         },
+      {
+         "name":"totalDailyMinutes",
+         "type":"int",
+         "doc":"Total Daily Minutes."
```

Comment:
Does this describe the number of minutes in a day?

Reply:
Compatible Garmin devices accumulate intensity minutes throughout the day. This is what is available for current total daily minutes. This value indicates intensity minutes accumulated till that point of the day.

Reply:
Helpful video - https://www.google.com/search?q=garmin+fenix+6+intensity+minutes&rlz=1C5CHFA_enIN763IN763&oq=garmin+fenix+6+intensity+&aqs=chrome.0.0j69i57j0.7869j0j4&sourceid=chrome&ie=UTF-8#kpvalbx=_eBPaXqfIHtmP4-EP3omL6Ak41